### ISSUE TITLE
UUIDs unsupported

### ISSUE THREAD

**Issue by boardfish** [nulldb/nulldb#98]
I'm using PostgreSQL with UUIDs. These get configured as follows in the schema:

```ruby
...
  enable_extension "pgcrypto"
  enable_extension "plpgsql"

  create_table "things_that_belong_to_users", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
    t.uuid "user_id", null: false
...
```

This is responsible for the following error:

```
undefined method `uuid' for #<ActiveRecord::ConnectionAdapters::NullDBAdapter::TableDefinition:0x0000556d72dcd3b0>
```

This seems to suggest `uuid` should be stubbed.

**Comment by cabello** [comment]
Fixed by #97 I will release a new version to Rubygems soon.

**Comment by cabello** [comment]
Done: https://rubygems.org/gems/activerecord-nulldb-adapter/versions/0.5.1
